src/Jackett/Indexers/BitSoup.cs

```diff
@@ -0,0 +1,284 @@
+        }
+
+        public async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken configJson)
+        {
+            configData.LoadValuesFromJson(configJson);
+            if (configData.AlternateLink.Value != null && configData.AlternateLink.Value != "")
+            {
+                if (!configData.AlternateLink.Value.EndsWith("/"))
+                {
+                    configData.AlternateLink.Value = null;
+                    throw new Exception("AlternateLink must end with a slash.");
+                }
+                var match = Regex.Match(configData.AlternateLink.Value, "^https?:\\/\\/(?:[\\w]+\\.)+(?:[a-zA-Z]+)\\/$");
+                if (!match.Success)
+                {
+                    configData.AlternateLink.Value = null;
+                    throw new Exception("AlternateLink must be a valid url.");
+                }              
+            }
+            var pairs = new Dictionary<string, string> {
+                { "username", configData.Username.Value },
+                { "password", configData.Password.Value },
+                
+            };
+
+            var loginPage = await RequestStringWithCookies(UseLink, string.Empty);
+
+            var result = await RequestLoginAndFollowRedirect(LoginUrl, pairs, loginPage.Cookies, true, null, LoginReferer, true);
+            await ConfigureIfOK(result.Cookies, result.Content != null && result.Content.Contains("logout.php"), () =>
+            {
+                CQ dom = result.Content;
+                var messageEl = dom["body > table.statusbar1 > tbody > tr > td > table > tbody > tr > td > table > tbody > tr > td"].First();
+                var errorMessage = messageEl.Text().Trim();
+                throw new ExceptionWithConfigData(errorMessage, configData);
+            });
+            return IndexerConfigurationStatus.RequiresTesting;
+        }
+
+        public async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
+        {
+            var releases = new List<ReleaseInfo>();
+            var searchString = query.GetQueryString();
+            var searchUrl = BrowseUrl;
+            var trackerCats = MapTorznabCapsToTrackers(query);
+            var queryCollection = new NameValueCollection();
+
+            
+            
+            if (!string.IsNullOrWhiteSpace(searchString))
+            {
+                queryCollection.Add("search", searchString);
+                queryCollection.Add("incldead", "0");
+                queryCollection.Add("cat", "0");
+                // Tracker cannot search multi categories
+                // so we either search "all"
+                // or do multiple searches
+                if (trackerCats.Count == 0)
+                {
+                    searchUrl += "?" + queryCollection.GetQueryString();
+                    await ProcessPage(releases, searchUrl);
+                } else
+                {
+                    foreach (var cat in trackerCats)

```

Comment:
A little concerned that if you search for TV or a Movie generically its going to hit the server 5 times.  If you are searching for something I would go for everything as the odds are you will get the item you are searching for then any extraneous items not in the correct category will be removed by the parent controller which calls this.